crates/aptos/src/node/mod.rs

```diff
@@ -231,6 +235,59 @@ impl ValidatorNetworkAddressesArgs {
     }
 }
 
+#[derive(Parser)]
+pub struct GetPoolAddress {
+    #[clap(long)]
+    pub(crate) owner_address: AccountAddress,
+    #[clap(long)]
+    pub(crate) operator_address: Option<AccountAddress>,
+    #[clap(long)]
+    pub(crate) employee_account_index: Option<u64>,
+    #[clap(flatten)]
+    pub(crate) rest_options: RestOptions,
+    #[clap(flatten)]
+    pub(crate) profile_options: ProfileOptions,
+}
```

Comment:
These need documentation / AccountAddress inputs should use load_account_arg

Reply:
Updated